src/main/java/com/iexec/commons/poco/chain/DealParams.java

```diff
@@ -117,19 +117,13 @@ public class DealParams {
     @JsonProperty("iexec_secrets")
     Map<String, String> iexecSecrets = Collections.emptyMap();
 
-    //Should be set by SDK
-    @JsonProperty("iexec_tee_post_compute_image")
-    String iexecTeePostComputeImage;
-
-    //Should be set by SDK
-    @JsonProperty("iexec_tee_post_compute_fingerprint")
-    String iexecTeePostComputeFingerprint;
```

Comment:
Isn't that dangerous? It may break some deals that have one of these properties. IMHO, it should be released in a major version.

Reply:
It was never documented or even supported in SDK.

Reply:
Maybe we can keep the property as is in `DealParams` but how will we decide when to remove them ?

Reply:
See last 2 commits, properties kept in `DealParams`